### DISCUSSION TITLE
How to get scores associated with an observation?

### DISCUSSION THREAD

**Discussion by pedroallenrevez** [langfuse/langfuse#4658] [Support]
I see nothing in the python API. Regular API only allows fetching all scores, configId doesn't seem to work. Moreover one can't provide traceId or observationId to get scores associated with them. 
Observations from python sdk, also don't come with scores, nor an id for a score, nothing. 

Seems like something that shold obviously be easy, I mean why monitor if you cant use the feedback from the platform? 
What am I missing here

**Comment by alanmun** [comment]
Did you ever figure this out Pedro? I am trying to do the same thing. My goal is to essentially reject overwrites of scoring, so that only the first ever score for some score config is accepted on the trace.

**Reply by alanmun** [reply]
Update, looks like this will work:

```python
    result = langfuse.fetch_trace(trace_with_score_id).data
    print(result.scores)
```

You'll need to get ahold of your trace's id for this though

**Reply by uahmad235** [reply]
This only gives score ids. How to fetch the whole object

**Reply by KripaMishra** [reply]
Use this 

```
trace = langfuse.api.trace.get("TraceID")
print(trace.scores)
```
